src/core/model.jl

```diff
@@ -181,6 +181,9 @@ end
 """
     nagents(model::ABM)
 Return the number of agents in the `model`.
+
+Example usage in:
+- [Wright-Fisher model of evolution](@ref): [sample!](@id neutral)
```

Comment:
I'm not sure I agree with this change. It essentially makes the docstring of a very simple function twice as large... Perhaps for `sample!` it does make sense to cross-ref ton an example, but for `nagents`? Not sure.

Reply:
I'll perhaps do all the complex ones, yeah. This is essentially just a test to make sure things work out as I suspect they should. But I'll dump it later - sure.